modules/ztp-monitoring-policy-deployment-progress.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-configuring-managed-clusters-policies.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-monitoring-policy-deployment-progress_{context}"]
+= Monitoring managed cluster policy deployment progress
+
+The ArgoCD pipeline uses `PolicyGenTemplate` CRs in Git to generate the {rh-rhacm} policies and then sync them to the hub cluster. You can monitor the progress of the managed cluster policy synchronization after the assisted service installs {product-title} on the managed cluster.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in to the hub cluster as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. The {cgu-operator-first} applies the configuration policies that are bound to the cluster.
++
+After the cluster installation is complete and the cluster becomes `Ready`, a `ClusterGroupUpgrade` CR corresponding to this cluster, with a list of ordered policies defined by the `ran.openshift.io/ztp-deploy-wave annotations`, is automatically created by the {cgu-operator}. The cluster's policies are applied in the order listed in `ClusterGroupUpgrade` CR.
++
+You can monitor the high-level progress of configuration policy reconciliation using the following commands:
++
+[source,terminal]
+----
+$ export CLUSTER=<clusterName>
+----
++
+[source,terminal]
+----
+$ oc get clustergroupupgrades -n ztp-install $CLUSTER -o jsonpath='{.status.conditions[?(@.type=="Ready")]}'
```

Comment:
- Please append | jq, otherwise the output is fairly unreadable. 

`oc get clustergroupupgrades -n ztp-install $CLUSTER -o jsonpath='{.status.conditions[?(@.type=="Ready")]}' | jq`

- From 4.12, type Ready will be removed. Should use type=Succeeded for 4.12.

Reply:
"From 4.12, type Ready will be removed. Should use type=Succeeded for 4.12." is there a JIRA tracking this change?

Reply:
@jc-rh is this tracked? 
If so, I think we don't need to worry about this PR. Right @aireilly ?

Reply:
Best to use this jsonpath which works with both the ready condition in 4.10/4.11 and the multiple conditions in 4.12.
`".status.conditions[-1:]"`

Reply:
Also, in 4.11/4.12 we added the state column which will print the reason message of the last condition:

```
$ oc get cgu -A
NAMESPACE     NAME           UPGRADE STATE      AGE
ztp-install   cnfde10-sync   UpgradeCompleted   226d
ztp-install   cnfde21        UpgradeCompleted   80d

```